.eslintrc.cjs

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  plugins: [
+    'vue',
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:vue/vue3-essential',
+  ],
+  parserOptions: {
+    parser: '@typescript-eslint/parser',
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+  rules: {
+    'no-undef': 'off',
+  },
+  overrides: [
+    {
+      files: [
+        '**/layouts/**/*.vue',
+        '**/pages/**/*.vue',
+        '**/error.vue',
+      ],
```

Comment:
Does it mean components are not tested? I guess the Vue plugin doesn't perfectly fit the Dust setup, right?

Reply:
Not is just means that for files that match these patterns the `vue/multi-word-component-names` is turned off.